multigen/pipes.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, model_id: str,
                         self.pipe = StableDiffusionPipeline.from_single_file(model_id, **args)
                     except TypeError as e:
                         pass
-                    self.pipe = StableDiffusionXLPipeline.from_single_file(model_id, **args)
+                        self.pipe = StableDiffusionXLPipeline.from_single_file(model_id, **args)
```

Comment:
Do we need `pass` here if there is some code after it?

Reply:
No, i removed it